subscriptions/conf.py

```diff
@@ -24,9 +24,10 @@ class Currency():
             n_sep_by_space (bool): Whether the currency symbol is
                 separated from negative values by a space.
             mon_decimal_point (str): The character used for decimal
-                points.
+                points. Use 0 if the currency doesn't need fraction.
             mon_thousands_sep (str): The character used for separating
-                groups of numbers.
+                groups of numbers. Use 0 if the currency doesn't need
```

Comment:
Sorry, I am not sure I am understanding this change. This argument refers to which separator is used for visually separating groups of numbers and not actually used in any fractional/decimal sense.

For example, we use `,` in USD to represent `$1,000,000.00`

I think we can just remove these changes?

Reply:
I thought that was for international format
Yeah that's not needed actually